### ISSUE TITLE
SA1013 should not warn when pattern case

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/StyleCopAnalyzers#3053]
```cs
private static void M(InvocationExpressionSyntax invocation)
{
    switch (invocation)
    {
        case { Expression: MemberAccessExpressionSyntax { Expression: { } expression } }:
            break;
    }
}
```

![image](https://user-images.githubusercontent.com/1640096/68126267-b7863c00-ff13-11e9-8d67-57c0b9cfc0a2.png)

**Comment by JohanLarsson** [comment]
https://github.com/dotnet/roslyn/issues/39683#issuecomment-549598612
